src/store-api/src/storage/engine.rs

```diff
@@ -82,11 +84,19 @@ pub struct EngineContext {}
 pub struct CreateOptions {
```

Comment:
Looks like `CreateOptions` is identical to `OpenOptions`, can we merge them?

Reply:
Maybe we need to keep them to identify the difference between "create and open" operation and "open if exist" operation.

Reply:
Looks like we can move the common options into a new struct named `TableConfig` (or some similar names) and share it between the two. Or just reuse existing `TableOptions`?